frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAddTemplate.tsx

```diff
@@ -140,6 +141,15 @@ const CustomServingRuntimeAddTemplate: React.FC<CustomServingRuntimeAddTemplateP
                 onClick={() => {
                   setIsLoading(true);
                   // TODO: Revert back to pass through api once we migrate admin panel
+                  if (!isServingRuntimeKind(YAML.parse(code))) {
+                    setError(
+                      new Error(
+                        'The "kind" should be "ServingRuntime", and the following fields are required: container, supportedModelFormats.',
```

Comment:
@DaoDaoNoCode the required field is `containers` (plural) 
```suggestion
                        'The "kind" should be "ServingRuntime", and the following fields are required: containers, supportedModelFormats.',
```